velox/type/HugeInt.h

```diff
@@ -52,6 +52,32 @@ class HugeInt {
       char* serializedData) {
     memcpy(serializedData, &value, sizeof(int128_t));
   }
+
+  static FOLLY_ALWAYS_INLINE int128_t stringToInt128(const std::string& str) {
```

Comment:
nit: Rename it to `parse`?  Also put the implementation to cpp as there is no benefit inlining it.

Reply:
Also add some tests for it please

Reply:
Fixed